### ISSUE TITLE
Safari has weird colors in dark mode

### ISSUE THREAD

**Issue by EricAndrechek** [EricAndrechek/FermiQuestions#1]
While in Safari (only tested on mobile safari on iPhone) some of the link and button colors appear as orange instead of blue. Additionally, the text box to type the answers in sometimes appears as white. 

CSS will need to be looked at and potentially a new hacky solution put in place specifically for Safari's rendering engine.

**Comment by Haadi-Khan** [comment]
Potentially helpful information:

I use the hyperweb extension for safari on my phone, and it has a dark theme plugin built in. If I toggle dark mode on the site, and also have the dark mode from hyperweb on, it switches from the orange color to the blue you would expect. Maybe just hard setting some of the css (aka just the colors) instead of relying in the import could be an easy fix. Given that hyperweb is just adding additional css I don't think it'd be too hard. I can't readily test safari though, so I don't know how much help I can be. 

Side note, as you'd expect hyperweb + site light theme works well and renters just how you'd expect.

**Comment by EricAndrechek** [comment]
Yeah, I suppose since the codebase is small and minimal I could probably just hard code the CSS values to make those changes rather than attempt to apply them automatically.
